content/articles/facilitate-media-queries-using-scss-mixins/index.md

```diff
@@ -0,0 +1,154 @@
+
+```scss
+// We define the mixins for two major breakpoints mobile and tablet
+ 
+ // $breakpoint is the variable that can have several values
+@mixin devices ($breakpoint) { //the name of the mixin is devices
+  
+    @if $breakpoint==tablet {    
+    @media only screen and (max-width: 680px) {
+      @content;
+    }
+  }
+ 
+  @if $breakpoint==mobile {
+    @media only screen and (max-width: 481px) {
+      @content;
+    }
+  }
+}
+ 
+```
+
+The above block of code shows how mixins are mapped out and associated with names.
+
+### STEP 3
+#### Media queries using SCSS mixins
+
+As you can see in the snippet below, the styling can be affected by simply calling up the mixins. From what has been defined, the color changes appropriately for tablet and mobile screen sizes.
+
+```scss
+// Handling the responsiveness
+.responsive {
+    background: yellow;
+  // The normal background color is set to yellow
+ 
+  @include devices(tablet) {
+        background: lightblue;
+        // code set for tablet
+    }
+ 
+  @include devices(mobile) {
+        background: aquamarine;
+        h1{
+            color: red;
+        }
+        // code set for mobile    
+    }
+}
+ 
+```
+
+By simply calling up the mixins created, you can apply stylings as you wish. These SCSS code will compile to generate the corresponding CSS code:
+ 
+```CSS
+.responsive { 
+  background: yellow;
+}
+
+@media only screen and (max-width: 680px) {
+  .responsive {
+    background: lightblue;
+  }
+}
+
+@media only screen and (max-width: 481px) {
+  .responsive {
+    background: aquamarine;
+  }
+  .responsive h1 {
+    color: red;
+  }
+}
+```
+
+The result is a short block of CSS code generated automatically, with styling for the breakpoints defined.
+
+Now, you can simply use the mixins where you choose and alter the media queries by just editing the mixins. The resulting CSS code are clearly neater, without the long list of breakpoints. When you run the code and resize the screen to the various widths, you will observe how the background colour changes in correspondence.
+
+### Conclusion
+You have successfully acquired guidelines on how to install the SASS compiler, learnt how to set up the SCSS mixins and then utilized the best technique that uses SCSS mixins to build a website that would fit in appropriately and display perfectly across all devices.
+
+With the SCSS mixins, you don't have to bury a long list of breakpoints in your code. Instead, you can have them in a centralized location for easy management and better responsiveness.
+
+I hope you found this article helpful in building responsive websites quickly and efficiently.
+
+Thanks to SCSS, you can see CSS media queries in a less intimidating light. 
+
+### Further reading
+- [The easier way to write media queiries](https://levelup.gitconnected.com/the-easier-way-to-write-media-queries-with-scss-mixin-c7c956150551)
+- [Mixin documentation](https://sass-lang.com/documentation/at-rules/mixin)
```

Comment:
For the items in this section, please name them as follows: 
`Title - Source`
For example:
`Writing Media Queries with Sass Mixins - Itnext.io`, etc.

Reply:
Changed the title format for Further Reading as directed. Thank you